feature/session/src/main/java/io/github/droidkaigi/confsched2020/session/ui/item/SpeakerDetailItem.kt

```diff
@@ -48,12 +36,26 @@ class SpeakerDetailItem @AssistedInject constructor(
 
         speaker.imageUrl ?: onImageLoadedCallback()
 
+        val context = viewBinding.root.context
+        val placeHolder = VectorDrawableCompat.create(
```

Comment:
Thanks!
Can you use lazyWithParam method like this :+1:

```kotlin
    private val placeholder by lazyWithParam<Context, VectorDrawableCompat> { context ->
        VectorDrawableCompat.create(
            context.resources,
            R.drawable.ic_person_outline_black_32dp,
            null
        )?.apply {
            setTint(
                ContextCompat.getColor(context, R.color.speaker_icon)
            )
        }
    }
```

Reply:
Thanks for your advice. I will try this code:smile: